Source/WebKit/WebProcess/WebPage/Cocoa/WebPageCocoa.mm

```diff
@@ -474,6 +474,21 @@
     completionHandler({ span(accessibilityRemoteTokenData().get()) }, getpid());
 }
 
+void WebPage::resolveAccessibilityHitTestForTesting(const WebCore::IntPoint& point, CompletionHandler<void(String)>&& completionHandler)
+{
+#if PLATFORM(MAC)
+    if (id coreObject = [m_mockAccessibilityElement accessibilityRootObjectWrapper]) {
+        if (id hitTestResult = [coreObject accessibilityHitTest:point]) {
+            ALLOW_DEPRECATED_DECLARATIONS_BEGIN
+            completionHandler([hitTestResult accessibilityAttributeValue:@"AXInfoStringForTesting"]);
+            ALLOW_DEPRECATED_DECLARATIONS_END
+            return;
+        }
+    }
+#endif
+    completionHandler(makeString("NULL"_s));
```

Comment:
I'm surprised there are no compiler warnings about `point` being unused on `!PLATFORM(MAC)` 🤔 Would've thought we needed an `UNUSED_PARAM(point)`

Reply:
There probably will be 😅 , I've just been fixing other errors on `!PLATFORM(MAC)`. So I'll add the unused param in.